kube-monitoring/charts/values.yaml

```diff
@@ -206,31 +206,27 @@ kubeMonitoring:
     pods:
       enabled: true
 
-      ## Endpoints of the selected pods to be monitored
-      ## https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#podmetricsendpoint
+      ## Monitor Pods with the following port name
+      port: metrics
+
+      ## RelabelConfigs to apply to samples before scraping
+      ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
+      ##
+      additionalRelabelings: []
+      # - sourceLabels: [__meta_kubernetes_pod_node_name]
+      #   separator: ;
+      #   regex: ^(.*)$
+      #   targetLabel: nodename
+      #   replacement: $1
+      #   action: replace
+
+      ## MetricRelabelConfigs to apply to samples after scraping, but before ingestion.
+      ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
       ##
-      podMetricsEndpoints:
-          ## Monitor Pods with the following port name
-        - port: metrics
```

Comment:
Why did you want to move this?

Reply:
The annotations selection is a bit hidden. https://github.com/cloudoperators/greenhouse-extensions/blob/2b01d25353046ddd90849b036a8f433297fdbd76/kube-monitoring/charts/templates/podmonitor-sd.yaml#L15

Reply:
ok then I've parsed it correctly 👍